### PR TITLE
Use helper for mandatory relation pairs

### PR DESCRIPTION
**Pull request by sed-i**
## Issue
Mandatory relation pairs were modeled incorrectly.

## Solution
Use helper for mandatory relation pairs.

In tandem with:
- https://github.com/canonical/cos-lib/pull/25
- https://github.com/canonical/grafana-agent-operator/pull/85

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -4,7 +4,8 @@
 # FIXME: Packing the charm with 2.2.0+139.gd011d92 will not include dependencies in PYDEPS key:
 # https://chat.charmhub.io/charmhub/pl/wngp665ycjnb78ar9ojrfhxjkr
 # That's why we are including cosl here until the bug in charmcraft is solved
-cosl
+#cosl
+git+https://github.com/canonical/cos-lib@feature/mandatory-relation-pairs
```
Comment (Abuelodelanada) [inline requirements.txt:8]: Why do not we wait for cosl to be released to pypi so we do not have to create another PR ?

**Review by Abuelodelanada — APPROVED** [review]

File: requirements.txt (hunk 1 above)
Comment (sed-i) [reply requirements.txt]: Yep, done!
